Common/Configurations/TokenManagement.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+
+namespace Common.Configurations
+{
+    [JsonObject("tokenManagement")]
+    public class TokenManagement
+    {
+        [JsonProperty("secret")]
+        public string Secret { get; set; }
+
+        [JsonProperty("issuer")]
+        public string Issuer { get; set; }
+
+        [JsonProperty("audience")]
+        public string Audience { get; set; }
+
+        [JsonProperty("accessExpiration")]
+        public int AccessExpiration { get; set; }
+
+        [JsonProperty("refreshExpiration")]
+        public int RefreshExpiration { get; set; }
+    }
```

Comment:
Do we need token settings in common?

Reply:
I think we shoud save all our configuration classes in one plase in progect, separately from Dtos